src/opm/input/eclipse/Parser/Parser.cpp

```diff
@@ -74,6 +74,22 @@
 #include <fmt/format.h>
 
 namespace {
+    /// \brief Whether a keyword is a global keyword and how many to skip
+    ///
+    /// Those are allowed before RUNSPEC.
+    /// \return false if this is not a global keyword or the number of keywords to skip
+    ///               otherwise (including itself.
```

Comment:
The documentation says that the function returns false or a number but it only returns `true` or `false`.

Reply:
Argh! Thanks for noticing. Fixed